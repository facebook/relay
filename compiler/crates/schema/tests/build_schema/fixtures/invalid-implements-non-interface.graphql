# expected-to-throw

type User {
  id: ID
}

type Page implements User {
  id: ID
}
