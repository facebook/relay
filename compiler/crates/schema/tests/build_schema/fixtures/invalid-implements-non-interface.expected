==================================== INPUT ====================================
# expected-to-throw

type User {
  id: ID
}

type Page implements User {
  id: ID
}
==================================== ERROR ====================================
Expected an interface type for name 'User', got 'Object(0)'.
